[2024-09-12T02:28:41.492+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:28:41.531+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-12T02:28:41.564+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-12T02:28:41.566+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:28:41.606+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2024-09-07 00:00:00+00:00
[2024-09-12T02:28:41.621+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'task1', 'backfill__2024-09-07T00:00:00+00:00', '--job-id', '21872', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpzkx6i03e']
[2024-09-12T02:28:41.629+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15445) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:28:41.635+0000] {standard_task_runner.py:72} INFO - Started process 15469 to run task
[2024-09-12T02:28:41.632+0000] {standard_task_runner.py:105} INFO - Job 21872: Subtask task1
[2024-09-12T02:28:41.763+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-09-07T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:28:41.906+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v02' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-09-07T00:00:00+00:00'
[2024-09-12T02:28:41.911+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:28:41.914+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:28:41.916+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-09-12T02:28:41.942+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:28:41.943+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-09-12T02:28:41.947+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:28:42.021+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:28:42.022+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=task1, run_id=backfill__2024-09-07T00:00:00+00:00, execution_date=20240907T000000, start_date=20240912T022841, end_date=20240912T022842
[2024-09-12T02:28:42.072+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:28:42.086+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-12T02:28:42.094+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
